src/tests/fs_tests.rs

```diff
@@ -31,7 +31,15 @@ pub mod fs_tests {
         ut_lind_fs_fstat_complex();
         ut_lind_fs_getuid();
         ut_lind_fs_load_fs();
-        ut_lind_fs_mknod();
+
+        // mknod_syscall_tests
+        ut_lind_fs_mknod_empty_path();
```

Comment:
I think these may move to the new test format, but I'll let @yashaswi2000 weigh in...

Reply:
Closing this conversation based on the discussion today. Leaving the tests in this format as of now for this PR.